plugins/agent-utilities/skills/fusion-parametric-design/tests/test_pipeline_seams.py

```
@@ -700,5 +1019,219 @@ def test_the_cli_exits_two_and_says_so(self) -> None:
+    sums the blocks of a candidate group and asks the router whether they are
+    swept by one rotation about this axis. Nothing here is a fixture: the mesh is
+    segmented, fitted, planned and emitted, and the emitted script runs against
+    the Fusion doubles.
+    """
+
+    #: The acceptance run's own plate: 80 x 50 x 10, whose only coaxial outward
+    #: turned surface is one round of radius 10 that is not centred on it.
+    ACCEPTANCE = dict(
+        width=80.0, depth=50.0, height=10.0, post_radius=10.0, post_height=6.0,
+        post_centre=(22.0, 16.0), nx=16, ny=12, nz=5,
+    )
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)